rpa-home/modules/ROOT/pages/invoke-rpa-from-flow.adoc

```diff
@@ -0,0 +1,17 @@
+# Invoke RPA Services from Salesforce Flow
+
+You can run RPA processes as part of a Salesforce Flow by following these steps:
+
+. xref:invoke-rpa-from-flow-project.adoc[Create an automation project.]
+. xref:invoke-rpa-from-flow-build.adoc[Build your process in RPA Builder.]
+. xref:invoke-rpa-from-flow-publish.adoc[Publish an invocable run configuration to Salesforce in RPA Manager.]
+. xref:invoke-rpa-from-flow-run.adoc[Call the Invocable Actions from Salesforce Flow.]
+
+## Before You Begin
+
+* If you use the one of the following subscription plans, refer to this documentation:
+** Automation Starter
```

Comment:
Where is this documentation? Can you link to it? If it is Salesforce documentation, you can say "refer to this Salesforce documentation."

Reply:
'This documentation' refers the docs the user is reading at this moment. 
If he uses another subscription plan, he has to read (the linked) other docs.